components/eventing-controller/README.md

```diff
@@ -2,8 +2,9 @@
 
 ## Overview
 
-This component contains controllers for various CustomResourceDefinitions related to eventing in Kyma. The controller comes with this container:
-- [controller](https://github.com/kubernetes-sigs/controller-runtime)
+This component contains controllers for various CustomResourceDefinitions related to eventing in Kyma. The following controllers come with this container:
+- [controller](https://github.com/kyma-project/kyma/blob/master/components/eventing-controller/cmd/eventing-controller/main.go) which lays down the eventing infrastructure in BEB
```

Comment:
The component is named Eventing Controller and includes two controllers: `controller` and `nats-controller`. It that is the naming convention we want to take, let's stick to it throughout the doc as later in the docs, I can see `controller` is referred to as Eventing Controller which is quite misleading. Also, let's use either `nats-controller` or NATS (NATS, not Nats, right?) Controller:
![image](https://user-images.githubusercontent.com/40655785/104023731-5b2b4600-51c2-11eb-9428-0a1f07d88c3a.png)

Reply:
@kazydek You're definitely right. IMHO, the best approach is to change the code/directories of `controller` to be `beb-controller` and change the documentation then based on that.
Is it fine if we resolve this for now and create a separate ticket for renaming the code occurrences as well as the docs, as it will be a technical task with intensive technical testing for this breaking change?
CC: @sayanh

Reply:
It makes sense to change the names to be more descriptive in the future. For now, however, if we have the distinction between `controller` and `nats-controller`, I'd suggest sticking to it throughout the doc. I already added comments that change it :)